app/presenters/transaction_file_body.presenter.js

```diff
@@ -38,7 +38,7 @@ class TransactionFileBodyPresenter extends BasePresenter {
       col17: '',
       col18: '',
       col19: '',
-      col20: data.chargeValue,
+      col20: this._signedCreditValue(data.chargeValue, data.chargeCredit),
```

Comment:
If my previous suggestion is taken on this would become

```suggestion
      col20: this._reverseSignOfValue(data.chargeValue),
```